js/compressed-token/src/idl/light_compressed_token.ts

```diff
@@ -2808,108 +2723,23 @@ export const IDL: LightCompressedToken = {
     errors: [
         {
             code: 6000,
-            name: 'PublicKeyAmountMissmatch',
-            msg: 'public keys and amounts must be of same length',
+            name: 'SignerCheckFailed',
+            msg: 'Signer check failed',
         },
         {
             code: 6001,
-            name: 'SignerCheckFailed',
-            msg: 'SignerCheckFailed',
+            name: 'CreateTransferInstructionFailed',
+            msg: 'Create transfer instruction failed',
         },
         {
             code: 6002,
-            name: 'ComputeInputSumFailed',
-            msg: 'ComputeInputSumFailed',
+            name: 'AccountNotFound',
+            msg: 'Account not found',
         },
         {
             code: 6003,
-            name: 'ComputeOutputSumFailed',
```

Comment:
The IDL change looks a bit fishy, probably the reason why js jobs fail

Reply:
js jobs failed in post setup lol

Reply:
thanks for reviewing!